src/commands/mod.rs

```diff
@@ -1,25 +1,21 @@
 mod publish;
 
 use crate::commands::publish::Publish;
+use enum_dispatch::enum_dispatch;
+use anyhow::Result;
 use structopt::StructOpt;
 
 /// Available commands
+#[enum_dispatch(Run)]
 #[derive(StructOpt, Debug)]
 pub enum Command {
     /// Publishes this mod to BeatMods2
     Publish(Publish),
 }
 
 /// Run function, the trait is not really needed but it's a nice convention
```

Comment:
This doc comment isn't really accurate if enum_dispatch is used

Reply:
It's now accurate again since I'm no longer using enum_dispatch (and there's no requirement of a trait for the macro).